### PR TITLE
updated nav and added pages directory plus about markdown and sample …

### PR DESCRIPTION
**Pull request by LauraHilliger**
…content - I'm going to merge and see what I'm working with!

### CODE REVIEW

**Comment by LauraHilliger** [comment]
Oh boo! @semioticrobotic I don't have write access here. I was just going to see if I could start figuring this out...

**Comment by semioticrobotic** [comment]
Well that stinks! And I even double checked to make sure you did. I will never understand GitHub permissions! Let me try again.

**Comment by semioticrobotic** [comment]
Okay! Try now please (and sorry again).

**Review by semioticrobotic — APPROVED** [review]
All looks good to me! Eager to see how the `splash` layout looks here.
